.github/workflows/release.yml

```diff
@@ -0,0 +1,35 @@
+name: goreleaser
+
+on:
+  push:
```

Comment:
Shouldn't this be `release` with a configured type? e.g. 

```
on:
  release:
    types:
      - created
```

Reply:
Potentially, yes.  Still working out what will be the best way to pull this off.  I'm certainly open to suggestions if you've done this kind of thing before.

Reply:
In this instance, I would definitely use `release` instead of `push` since this action only needs to run for a release and not every time there is a push into the main branch.

Reply:
That's fair.  There's less ambiguity there as well.  I like it.